code/index.html

```diff
@@ -1,21 +1,46 @@
 <!DOCTYPE html>
-<html>
-
+<html lang="de">
+ 
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Carole & Tobias</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
     <!-- If you're using google fonts, that should be added here -->
+    <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@300&family=Playfair+Display:wght@500&display=swap" rel="stylesheet">  
   </head>
+
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+   
+    <img src="Hanging Flowers.png" alt="Flowers" class="image1">
+
+    <div class="nav">
+      <ul>
+        <li class="home"><a class="active" href="#">Home</a></li>
+        <li class="wann-wo"><a href="#">Wann & Wo</a></li>
+        <li class="Übernachtung"><a href="#">Übernachtung</a></li>
+        <li class="honeymoon.fund"><a href="#">Honeymoon Fund</a></li>
+        <li class="rsvp"><a href="#">RSVP</a></li>
+      </ul>
+    </div>
+    
+    
+    <h1>Carole & Tobias</h1>
+    <h2>Wir sagen Ja.</h2>
+
+  <center><img src="Carole&Tobias.jpeg" alt="Carole & Tobias" style="width:300px;height:400px;padding:0%"></center>
```

Comment:
We should try to keep the styling in the CSS file for better readability. To move this styling to the CSS-file, you could give this image a class name, like this:
`<img class="big-image" src="Carole&Tobias.jpeg" alt="Carole & Tobias">`
and then style the class like this:
```
.big-image {
  display: block;
  width: 300px;
  margin: 0 auto;
}
```
Note that I removed the `center`-tag and the `style`-attribute from your `img`-tag. Also that I removed the `height` property from the CSS, because that's causing the image to get a little squished together.